setup.py

```diff
@@ -4,6 +4,8 @@
 from setuptools import setup, find_packages
 from setuptools.command.test import test as testcommand
 
+VERSION = __import__('pylmod').VERSION
```

Comment:
I don't know what you're suggesting.  :-(  How do I "just use version"?  Am I setting the version value in __init__.py or in setup.py?